src/plugins/profiling/server/routes/search_flamechart.ts

```diff
@@ -112,96 +123,178 @@ export function registerFlameChartSearchRoute(router: IRouter<DataRequestHandler
         // eslint-disable-next-line no-console
         console.log('EventsIndex', eventsIndex);
 
+        // Using filter_path is less readable and scrollSearch seems to be buggy - it
+        // applies filter_path only to the first array of results, but not on the following arrays.
+        // The downside os `_source` is: it takes 2.5x more time on the ES side (see "took" field).
```

Comment:
```suggestion
        // The downside of `_source` is: it takes 2.5x more time on the ES side (see "took" field).
```

Minor correction: Is `of` the correct word? I wasn't sure.